Userland/Utilities/gunzip.cpp

```diff
@@ -65,17 +67,35 @@ int main(int argc, char** argv)
 
         auto input_stream_result = Core::InputFileStream::open_buffered(input_filename);
 
+        if (input_stream_result.is_error()) {
+            dbgln("Failed opening input file for reading: {}", input_stream_result.error());
```

Comment:
For command-line utilities, we should report errors to `stderr`, so `warnln()` rather than `dbgln()` :)

Reply:
i wanted to do that, but i noticed that one of the optional flags enables piping to stdout

Reply:
oh, nvm, warnln writes to stderr, so it should still be just fine, will fix :)